pittAPI.py

```diff
@@ -472,24 +472,24 @@ def get_dining_locations_by_status(self, status=None):
                             #    i.find("div").getText().replace(u'\u2013', '-').replace('\n', '')
                             #])
                             dining_locations[self._encode_dining_location(i.find("span").getText())] = {
-                                "name"   : i.find("span").getText(),
-                                "hours"  : i.find("div").getText().replace(u'\u2013', '-').replace('\n', '').replace('Next: ', ''),
-                                "status" : "closed" if "Next:" in i.find("div").getText() else "open"
+                                "name":i.find("span").getText(),
```

Comment:
Last minor nitpick then this is good to merge, but there should be a space after the colon between it and the value!